wp-cli/class-theme-check-cli.php

```diff
@@ -0,0 +1,157 @@
+<?php
+
+if ( defined( 'WP_CLI' ) && WP_CLI ) {
+	$parent_dir = dirname( __DIR__ );
+	require_once $parent_dir . '/checkbase.php';
+	require_once $parent_dir . '/main.php';
+	WP_CLI::add_command( 'theme-check', 'Theme_Check_Command' );
+}
+
+class Theme_Check_Command extends WP_CLI_Command {
+	/**
+	 * Run a theme check on the specified theme or the current theme.
+	 *
+	 * @param array $args       Indexed array of positional arguments.
```

Comment:
These 3 lines are breaking the expected doc format of WP-CLI command. So format validation is not working. You can confirm it by checking `wp help theme-check run`. Currently you wont see arguments properly. When you remove these lines, it will work as expected.

Reply:
You can move these after Examples.

Reply:
Good. Done here: https://github.com/WordPress/theme-check/pull/458/commits/e998dc02035639c6f0bb3fa297842a09391abd95